doc/introduction/installation.rst

```diff
@@ -88,6 +88,9 @@ will error out in the configure if their required
 dependencies are missing, but they will update the CMake cache variable so that
 the next configure will succeed (with that component disabled).
 
+
+.. _configuration:
```

Comment:
Since you're reworking the doc, I forgot to add Perfetto in the dependency table above when we added it.